modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -126,7 +128,25 @@ instead.
 `CountBadge` now uses `createComponent`, Canvas Tokens, and our new styling utilities. The component
 API has not been changed, and it should behave identically as it did in previous versions.
 
-### FormField Preview
+### Text
+
+`Text` has been refactored to use our new styling utilities, new tokens and no longer extends the
+`Box` component. The component API has not been changed and it should behave identically as it did
+in previous versions while still supporting Emotion `styled` and style props. Type level components:
+`Title`, `Heading`, `BodyText` and `Subtext`, have not been changed because they extend `Text`
+components and all changes made to `Text` will be applied to them as well.
+
+### Label Text
+
+`LabelText` has been refactored to use our new styling utilities, utilize new tokens and no longer
+extends the`Text` component. The component API is similar to the previous one and still supports
+`variant`, `disabled` and `Box` styles props. The `typeLevel` prop has been removed from `LabelText`
+and we suggest to use type level components, `Title`, `Heading`, `BodyText` or `Subtext`.
```

Comment:
```suggestion
`LabelText` has been refactored to use our new styling utilities and tokens and it no longer
extends the `Text` component. Because of this change, the `typeLevel` prop has been removed from `LabelText`
and we suggest to use type level components, `Title`, `Heading`, `BodyText` or `Subtext`. The component still supports `variant`, `disabled` and `Box` styles props.
```